.travis.yml

```diff
@@ -21,7 +21,8 @@ before_script:
   - ./configure --enable-ccache
 script:
   - make tidy
-  - make rustc-stage1 -j4
+  - make -j4
```

Comment:
Btw, why not just `make check -j4`?
With 4 jobs `check` is usually finished even before `all`, i.e. `time(make check -j4) ≈ time(make -j4)`.